prover/crates/lib/prover_fri_types/src/lib.rs

```diff
@@ -164,7 +166,52 @@ impl ProverJob {
 #[derive(Debug, Clone, Copy, Eq, PartialEq, Hash, serde::Serialize, serde::Deserialize)]
 pub struct ProverServiceDataKey {
     pub circuit_id: u8,
-    pub round: AggregationRound,
+    pub stage: ProvingStage,
+}
+
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Hash, serde::Serialize, serde::Deserialize)]
+pub enum ProvingStage {
```

Comment:
well, it lays deeply in how keystore is working. We have aggregation round struct, which is used nearly everywhere and it is also used for storing keys. Since compression/SNARK is not really part of aggregation(and I needed to expand the struct to have compression and snark there) it doesn't make sense to name it like that.